README.md

```diff
@@ -87,6 +87,22 @@ Show message with ID 1
 
     RETR 1
 
+## Use with docker-machine
+
+Because service clients, like the addon-administration, that are using the
+3-legged flow to authenticate a user with a redirect to OSIAM only know one
+endpoint to redirect to and back to themselves, this container will be working
```

Comment:
This container will only work when forwarding the port of the tomcat **that** is running inside the container.